packages/design-system-tokens/src/themes/medicare-components.ts

```diff
@@ -276,7 +276,7 @@ export const components: AnyTokenValues = {
     '__background-color':                         t.color['background'],
     '__border-color':                             t.color['border'],
     '-close__color':                              t.color['black'],
-    '-header__background-color':                  t.color['gray-lightest'],
+    '-header__background-color':                  t.color['transparent'],
```

Comment:
mgov Drawer header background is either white or transparent. Given the Drawer itself has a white background, I opted for the transparent value.